tests/Graph/WeightedShortestPathTest.cpp

```diff
@@ -435,7 +435,7 @@ TEST_P(WeightedShortestPathTest, shortest_path_A_F_outbound) {
     aql::TraversalStats stats = finder.stealStats();
     // We have to lookup the vertex
     // 4x vertices, 3x edges
-    EXPECT_EQ(stats.getScannedIndex(), 17U);
+    EXPECT_EQ(stats.getScannedIndex(), 16U);
```

Comment:
To be honest, this is not a very sensible test. The number subtly depends on the implementation of the algorithm, since it depends on the choice on which side we expand next. I simply adjustet the test to the new situation.